prombench/manifests/cluster-infra/2_ingress-nginx-controller.yaml

```diff
@@ -185,6 +197,30 @@ subjects:
 
 ---
 
+kind: Service
```

Comment:
nit: add in PR description why this may have worked in the past.

Reply:
Done.